src/tests/baseservices/exceptions/stackoverflow/stackoverflowtester.cs

```diff
@@ -4,27 +4,45 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.IO;
+using System.Linq;
 using System.Text;
 
 namespace TestStackOverflow
 {
     class Program
     {
+        const string ThisProjectName = "stackoverflowtester";
         static string s_corerunPath;
         static string s_currentPath;
 
-        static bool TestStackOverflow(string testName, string testArgs, out List<string> stderrLines)
+        static Process GetNativeAotProcess(string testName, string testArgs)
         {
-            Console.WriteLine($"Running {testName} test({testArgs})");
-            List<string> lines = new List<string>();
-
             Process testProcess = new Process();
+            string executableExtension = TestLibrary.Utilities.IsWindows ? ".exe" : "";
+            testProcess.StartInfo.FileName = Path.Combine(s_currentPath, "native", $"{ThisProjectName}{executableExtension}");
```

Comment:
I'd rather we don't hardcode the directory structure with `native` and everything if we can help it. Would this work?

```suggestion
            testProcess.StartInfo.FileName = Environment.ProcessPath;
```